feat: add `sdf` sprites via additional apis (#1492)
urbica/martin: docs/src/sources-sprites.md

}
```

> ##### Coloring at runtime via Signed Distance Fields (SDFs)

Author: > I need to make this more clear in the docs, why one would choose the one or the other image format

@hiddewie (hope you don't mind the ping too much)
Could you have a look at the docs if there is something that I missed?

Reviewer: This looks great, thanks!

martin --sprite /path/to/sprite_a --sprite /path/to/other/sprite_b
```

To use generate Signed Distance Fields (SDFs) images instead, the `--make_sdf` Option can be added.

> This makes images be handled as a SDFs allowing their color to be set at runtime in the map redering engines.

Author: ```suggestion
SDF-Images allow their color to be set at runtime in the map redering engine.
```

  sources:
    # SVG images in this directory will be published as a "my_sprites" sprite source
    my_sprites: /path/to/some_dir
  # This tells Martin to handle images in directories as Signed Distance Fields (SDFs)

>   # Images are handled as a signed-distance field (SDF) allow their color to be set at runtime in the map redering enines.

Author: ```suggestion
  # SDF-Images allow their color to be set at runtime in the map redering engine.
```
